SimCalorimetry/EcalSimProducers/python/esCATIAGainProducer_cfi.py

```diff
@@ -0,0 +1,12 @@
+import FWCore.ParameterSet.Config as cms
+
+EcalCATIAGainRatiosRcd =  cms.ESSource("EmptyESSource",
+                                recordName = cms.string("EcalCATIAGainRatiosRcd"),
+                                firstValid = cms.vuint32(1),
+                                iovIsRunNotTime = cms.bool(True)
+                                )
+
+EcalCATIAGainRatios = cms.ESProducer("EcalCATIAGainRatiosESProducer",
+                                     ComponentName = cms.string('EcalCatiaGainProducer'),
+                                     CATIAGainRatio = cms.double(10.))
```

Comment:
There is a constexpr array gains[NGAINS] in the EcalConstants.h that defines the two CATIA gains. Why is that needed if the gain ratio is going to be stored in the ES?